README.rst

```diff
@@ -49,6 +49,88 @@ You can run all of the tox environments using the command ``tox``.
 
 Flocker is also tested using `continuous integration`_.
 
+Cloudformation Deployment
+-------------------------
+
+* Generate a Cloudformation template:
+
+```
+$ python ./admin/installer/cloudformation.py > flocker-cloudformation-template.json
```

Comment:
Upon discussing with @wallrj , we decided to move the README edits to a followup branch if needed. Since docs reorg branch output (http://cube.lukemarsden.net:8080/docker-integration/cloudformation.html) covers most of this content, we will evaluate any missing steps once that shapes up. Thanks!